core/modules/wildcard_match.cc

```diff
@@ -312,8 +316,8 @@ int WildcardMatch::DelEntry(int idx, wm_hkey_t *key) {
   return 0;
 }
 
-CommandResponse WildcardMatch::CommandAdd(
-    const bess::pb::WildcardMatchCommandAddArg &arg) {
+template <typename T>
+CommandResponse WildcardMatch::Add(const T &arg) {
```

Comment:
Why is it templatized? `T` seems to be the same type, for all the callers of this function.

Reply:
This is because of the three part nature of the original commit series...

> we can simplify the new code by removing the leftover template-ized internal functions